tsconfig.json

```diff
@@ -1,8 +1,113 @@
 {
   "compilerOptions": {
-    "rootDir": ".",
-    "composite": true
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "allowSyntheticDefaultImports": true,
+    "jsx": "preserve",
+    "target": "ES2018",
+    "module": "ESNext",
+    "lib": ["DOM", "ESNext", "ESNext.AsyncIterable"],
+    "declaration": true,
+    "isolatedModules": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "esModuleInterop": true,
+    /* Strict Type-Checking Options */
+    "strict": true,
+    /* Additional Checks */
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "importsNotUsedAsValues": "error",
+    /* Module Resolution Options */
+    "moduleResolution": "Node",
+    "resolveJsonModule": true,
+    "types": ["@types/jest", "@nuxt/types", "@nuxtjs/i18n", "@pinia/nuxt"],
+    "typeRoots": ["./frontend/typings", "./node_modules/@types"],
+    "paths": {
+      "~/*": ["./frontend/src/*"],
+      "~~/*": ["./frontend/*"],
+      "@openverse/*": ["./packages/*/src"]
```

Comment:
Directing this to the `src` directory instead of the package root bypasses the `main` and `types` declarations in the package's `package.json`, allowing us to avoid needing to build types for packages that are only used internally.